### ISSUE TITLE
Deleted TV Show-Files: Still showing as "Monitored" rather than "Unmonitored"

### ISSUE THREAD

**Issue by ManiMatter** [Sonarr/Sonarr#5868]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When you delete all files for a TV show, the status is still "Monitored".

The setting "Unmonitor Deleted Episodes" under Settings/Media Management/File Management is checked.

In consequence, the functionality that auto-tags monitored tv shows (4.0.0.587 / Aug 2 2023) tags too many TV Shows (https://github.com/Sonarr/Sonarr/pull/5843).

In Radarr, the corresponding feature works correctly; when deleting a movie, the movies becomes "Unmonitored".

### Expected Behavior

When the folder containing the TV show is deleted, and the above setting is checked, the TV show should become unmonitored.

### Steps To Reproduce

1. Tick the setting "Unmonitor Deleted Episodes" under Settings/Media Management/File Management
2. Download a TV Show
3. Delete all files from the file system
4. Check the status (Is "Monitored" and should be "Unmonitored")

### Environment

```markdown
- OS: 22.04
- Sonarr:4.0.0.588
- Docker Install: Yes
- Using Reverse Proxy:No
```

### What branch are you running?

Develop

### Trace Logs?

https://privatebin.net/?381bc6542f860729#A2pJQEagmGDseAGCt3SitkMKPRgYDTJMHPYAUmYyMALa

Downloaded The Mandalorian, First Episode; Once downloaded I deleted it, and refreshed the monitored movies.
-> Mandalorian still shows "Monitored"

**Comment by augustuen** [comment]
Do the episodes stay monitored, or does the series stay monitored? I'd expect the episodes to be unmonitored, but there's no automatic unmonitoring of the Series, in case you still want to grab future episodes that haven't been added yet.

**Comment by ManiMatter** [comment]
hi both,

thank you for your inputs. 

You are right; the overall series as well as the seasons stay "monitored", but the respective episode that was deleted becomes "unmonitored". I hadn't realized that concept and it makes sense to me - but it makes my problem a bit more challenging.

Let me re-formulate what I am trying to accomplish, maybe you have an idea how to achieve it.

I have two Sonarr Instances, once for 4k, and the other for 720p.
The episodes marked as "Monitored" on the 4k instance (and only those) should also be downloaded on the 720p instance.
Not all episodes within a series/episode are marked as "monitored", since I already deleted some and those are now marked as "unmonitored".
I don't want "unmonitored" to be grabbed again in the 720p instance (since I deleted them by purpose already on the 4k instance)

I have set up a List Import on the 720p instance to grab the series form he 4k instance.
However, it either grabs all episodes from the 4k instance (regardless whether marked as monitored or not), or nothing.

How can I grab the "unmonitored" episodes only? I wanted to achieve that by using "tags", since in the List-Import I can specify what tags to look out for. But it seems the tags are applied on series-level, and not on episode-level.

Any ideas how I can keep the two instances in sync?
